Dockerfile.cds-tools

```
@@ -11,13 +11,19 @@ RUN apt-get install -y \
   curl \
   git \
   gnupg \
-  imagemagick libmagickcore-dev libmagickwand-dev \
+  imagemagick \
   jq \
-  libcurl4 libcurl3-gnutls libcurl4-openssl-dev \
-  libpq-dev \
+  libcurl4-openssl-dev `# Needed to pull in libcurl4 for curl installation and enable to openssl TLS backend` \
+  libmagickcore-dev \
+  libmagickwand-dev \
+  libpq-dev `# Needed to provide libpq-fe.h header to build 'pg' gem native extensions` \
   software-properties-common \
   unzip \
-  zip
+  zip \
+  zlib1g-dev `# Fixes 'zlib is missing; necessary for building libxml2'` \
```

Does this happen when installing a gem?

---

I believe it is nokogiri extensions

---

If this is related to nokogiri then maybe this should be
```suggestion
  zlib1g-dev `# For nokogiri -- fixes 'zlib is missing; necessary for building libxml2'` \
```